apps/site-app/src/containers/Accounts/Redeem/RedeemWidget/index.tsx

```diff
@@ -104,8 +116,9 @@ const validateChars = (re: () => RegExp) => (value: string) => {
       }
     : null;
 }
-const validateAnswer = validateChars(invalidAnswer);
-const validateQuestion = validateChars(invalidQuestion);
+const validateAnswer = validateChars(invalidAnswer, 3);
+const validateQuestion = validateChars(invalidQuestion, 3);
+const validateMessage = validateChars(invalidQuestion, 0);
```

Comment:
Message does not have a minimum length, this fixes the error flash on submission too